src/types/transactions/builder.ts

```diff
@@ -50,12 +50,14 @@ export type ConstructTransaction<
   O extends Partial<TransactionBase & A> = {}
 > = DistributiveOverwrite<TransactionBase & A, O>;
 
+export type SuggestedParamsWithoutMinFee = Omit<SuggestedParams, 'minFee'>;
```

Comment:
I think it would be more backwards-compatible to do the reverse: introduce a new `SuggestedParamsWithMinFee` that `SuggestedParamsRequest` would produce. That way existing code referencing `SuggestedParams` will remain the same.

Note that this would only work if a `SuggestedParamsWithMinFee` can be viewed as a `SuggestedParams`,  which I think should work because the former has all the fields required of the latter, but we'd need to double check to be sure.